server/channels/api4/ip_filtering.go

```diff
@@ -20,7 +20,7 @@ func (api *API) InitIPFiltering() {
 }
 
 func ensureIPFilteringInterface(c *Context, where string) (einterfaces.IPFilteringInterface, bool) {
-	if c.App.IPFiltering() == nil || !c.App.Config().FeatureFlags.CloudIPFiltering || c.App.License() == nil || c.App.License().SkuShortName != model.LicenseShortSkuEnterprise {
+	if c.App.IPFiltering() == nil || !c.App.Config().FeatureFlags.CloudIPFiltering || c.App.License() == nil || !c.App.License().IsCloud() || c.App.License().SkuShortName != model.LicenseShortSkuEnterprise {
```

Comment:
I wonder if ` c.App.IPFiltering() == nil` should already indicate if IP filtering is available. 

However, that would require  additional code to initiate it when the license changes.

Reply:
I don't think it's worth the effort to change it. The ticket was opened about a sentry error from a self hosted instance for this feature which is feature flagged - ie, someone enabled this without knowing what it does and "shot themselves in the foot". For now I think we're covered by simply disallowing someone to do that in a self hosted context.